pepper-apis/src/main/java/org/broadinstitute/ddp/export/json/structured/PicklistQuestionRecord.java

```diff
@@ -41,6 +46,9 @@ public PicklistQuestionRecord(String stableId, List<SelectedPicklistOption> answ
                     this.optionDetails.add(details);
                 }
             }
+            if (!groupOptions.isEmpty())  {
+                this.selected.addAll(groupOptions.keySet());
```

Comment:
Hmmm, are we sure about this? The keys of the map is the group ids. But the group itself is not really an option that users can click right? Rather, the user is choosing the options inside the group?

Reply:
I do have that impression too but thats what DSM wants to make it easy to display in DSM.
Double checked with Simone.  Basically user selected something inside that group is the justification for now.

Reply:
Hmmm, I'm not entirely convinced... but let's see how it goes.